node-operators/block-producers/onboarding/account-setup.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Setup an Account to Produce Blocks
+description: Follow these step-by-step instructions on becoming eligible to produce blocks on Tanssi and Tanssi Appchains and setting up your account for rewards payouts.
+---
+
+# Set Up Your Account to Produce Blocks on Tanssi
+
+## Introduction {: #introduction }
+
+Before you can start producing blocks on Tanssi and Tanssi Appchains, you'll need to set up your account and establish your eligibility.
+
+You must have spun up a [block-producing node](/node-operators/block-producers/onboarding/run-a-block-producer){target=\_blank} to tackle the account setup steps in this guide.
+
+You'll need to set up your account by generating [session keys](https://wiki.polkadot.network/docs/learn-keys#session-keys){target=\_blank} and mapping those session keys to your account.  This account is the one to which delegators will choose to delegate and where your rewards will be distributed.
+
+To establish eligibility, you must delegate yourself as a block producer and meet the minimum bond requirements.
+
+By following the steps outlined in this guide, you'll learn everything you need to know to get started producing blocks within the Tanssi ecosystem.
+
+## Important Variables {: #important-variables }
+
+When establishing eligibility to produce blocks, there are a couple of variables to be aware of:
+
+- **Minimum self-delegation** - there is a minimum amount you must self-delegate to be considered eligible
+- **Number of active candidates** - the top candidates by stake that can actively produce blocks
+
+|          Variable           |                          Value                           |
+|:---------------------------:|:--------------------------------------------------------:|
+|   Minimum self-delegation   | {{ networks.dancebox.staking.min_self_del.dance }} DANCE |
+| Number of active candidates | {{ networks.dancebox.staking.top_eligible_candidates }}  |
+
+## Map an Account to Your Block Producer Node {: #map-account }
+
+The first step is a two-step process that generates [session keys](https://wiki.polkadot.network/docs/learn-keys#session-keys){target=\_blank} and maps the session keys to your account. Session keys are used to perform network operations, such as signing blocks. By mapping the session key to your account, you create an association between your account and your block-producing node.
```

Comment:
We could add a line explaining the difference between the two accounts and why it is important that they are separated, being one the account that holds the staked funds, has an identity on chain, etc and the session keys that "are used to perform network operations...."